tests/server/blueprints/cases/test_cases_views.py

```diff
@@ -102,3 +103,104 @@ def test_causatives(app, user_obj, institute_obj):
 
             # THEN it should return a page
             assert resp.status_code == 200
+
+
+def test_matchmaker_add(app, institute_obj, case_obj):
+    # GIVEN an initialized app
+    # GIVEN a valid user and institute
+
+    with app.test_client() as client:
+        # GIVEN that the user could be logged in
+        resp = client.get(url_for('auto_login'))
+        assert resp.status_code == 200
+
+        # WHEN accessing the case page
+        resp = client.post(url_for('cases.matchmaker_add',
+                                institute_id=institute_obj['internal_id'],
+                                case_name=case_obj['display_name']))
+        # page redirects in the views anyway, so it will return a 302 code
+        assert resp.status_code == 302
+
+
+def test_matchmaker_matches(app, institute_obj, case_obj, mme_submission):
+    # GIVEN an initialized app
+    # GIVEN a valid user and institute
+
+    with app.test_client() as client:
+        # GIVEN that the user could be logged in
+        resp = client.get(url_for('auto_login'))
+        assert resp.status_code == 200
+
+        # add MME submission to case object
+        store.case_collection.find_one_and_update(
+            {'_id' : case_obj['_id']},
+            {'$set' : {
+                'mme_submission' : mme_submission
+            }}
+        )
+        assert store.case_collection.find({'mme_submission':{'$exists' : True}}).count() == 1
+
+        current_app.config['MME_URL'] = 'http://localhost:9020'
```

Comment:
Soo, but only if there is no server alive? Good for a virtual machine I guess, but may not work on a dev station? ;-)

Reply:
What makes you think that on a local machine it actually works? Because it doesn't

Reply:
Hm, I was thinking if the developer already has a service running on 9020? But perhaps this will fail already before that?

Reply:
If you want to do the things properly then you should test using a server-service that dynamically gets a free port, such as I did when testing the controllers. For the purpose of this test (just showing that there is a views method with that name that you can call, you could write whatever string in that address and it won't affect the test). Which fake port do you like? :)

Reply:
Hm, tempting.. 😄 Perhaps just mention that in the test "given" statement: I don't think we need an issue for making a fake server-service etc.

Reply:
I wrote it this way now
# Given mock MME connection parameters
current_app.config['MME_URL'] = 'http://fakey_mme_url:fakey_port' 

It felt like there are already controllers that are checking well all the processes related to MME so I didn't add the the fake service here as well, but we could always do it later maybe?